docker/images/pinot/etc/jmx_prometheus_javaagent/configs/broker.yml

```diff
@@ -1,76 +1,90 @@
 rules:
-- pattern: "\"org.apache.pinot.common.metrics\"<type=\"BrokerMetrics\", name=\"pinot.broker.([^\\.]*?).authorization\"><>(\\w+)"
-  name: "pinot_broker_authorization_$2"
+- pattern: "\"org.apache.pinot.common.metrics\"<type=\"BrokerMetrics\", name=\"pinot.broker.(([^\\.]+?)#)*([^\\.]*?).authorization\"><>(\\w+)"
```

Comment:
1. I think this means that regular table name does not contain `#`, is this the case? 
2. For table name with `.`, there is a way to match it, we don't necessarily need to change it to `#`